app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def self.update_comments_counter(post_id)
+    post = Post.find(post_id)
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ Required ] I think it would be better if you used one of the [Active Rails callback](https://guides.rubyonrails.org/active_record_callbacks.html) e.g the `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to the database, then make the `update_comments_counter` method `private`. This ensures that the `comments_counter` field is incremented anytime a comment is added to a post.